appendix/stata-guide.tex

```diff
@@ -40,6 +40,26 @@
 
 \section{Using the code examples in this book}
 
+For some implementation portions where precise code is particularly important,
+we provided minimal code examples either in the book or on the DIME Wiki.
+All code guidance is software-agnostic, but code examples are provided in Stata.
```

Comment:
this sentence i would suggest keeping in the intro (and then mentioning there the details in the appendix)